forge-gui/res/cardsfolder/upcoming/polukranos_reborn_polukranos_engine_of_ruin.txt

```diff
@@ -0,0 +1,23 @@
+Name:Polukranos Reborn
+ManaCost:G G G
+Types:Legendary Creature Hydra
+PT:4/5
+K:Reach
+A:AB$ SetState | Cost$ 6 WP | Defined$ Self | Mode$ Transform | SorcerySpeed$ True | AILogic$ Always | SpellDescription$ Transform CARDNAME. Activate only as a sorcery.
+AlternateMode:DoubleFaced
+Oracle:Reach\n{6}{W/P}: Transform Polukranos Reborn. Activate only as a sorcery. ({W/P} can be paid with either {W} or 2 life.)
+
+ALTERNATE
+
+Name:Polukranos, Engine of Ruin
+ManaCost:no cost
+Colors:white,green
+Types:Legendary Creature Phyrexian Hydra
+PT:6/6
+K:Reach
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self,Creature.Other+nonToken+Hydra+YouCtrl | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME or another nontoken Hydra you control dies, create a 3/3 green and white Phyrexian Hydra creature token with reach and a 3/3 green and white Phyrexian Hydra creature token with lifelink.
+SVar:TrigToken:DB$ Token | TokenScript$ gw_3_3_phyrexian_hydra_reach | SubAbility$ TrigToken2
+SVar:TrigToken2:DB$ Token | TokenScript$ gw_3_3_phyrexian_hydra_lifelink
+DeckHas:Ability$Token & Type$Hydra
```

Comment:
`Ability$Token|LifeGain` ... remove Type stuff, that's already a given because the host card has Hydra Type

Reply:
Reworked the Deck$ tags, thanks!